src/content/docs/es/guides/cms/ghost.mdx

```diff
@@ -0,0 +1,294 @@
+---
+title: Ghost y Astro
+description: Agrega contenido a tu proyecto Astro usando Ghost como CMS
+type: cms
+stub: false
+service: Ghost
+i18nReady: true
+---
+
+import FileTree from '~/components/FileTree.astro'
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+[Ghost](https://github.com/TryGhost/Ghost) es un sistema de gestión de contenido de código abierto, headless y construido en Node.js.
+
+## Integración con Astro
+En esta sección, usaremos la [API de contenido de Ghost](https://ghost.org/docs/content-api/) para traer tus datos a tu proyecto Astro.
+
+### Prerrequisitos
+Para comenzar, necesitarás lo siguiente:
+
+1. **Un proyecto Astro** - Si aún no tienes un proyecto Astro, nuestra [guía de instalación](/es/install/auto/) te ayudará a comenzar en cuestión de minutos.
+
+2. **Un sitio Ghost** - Se supone que tienes un sitio configurado con Ghost. Si no, puedes configurar uno en un [entorno local.](https://ghost.org/docs/install/local/)
+
+3. **Llave API de contenido** - Puedes crear una integración en la sección `Configuración > Integraciones` de tu sitio. Desde allí puedes encontrar tu `Llave API de contenido`.
+
+
+### Configuración de credenciales
+
+To add your site's credentials to Astro, create an `.env` file in the root of your project with the following variable:
```

Comment:
Hey 👋 , I think this line has to be removed as is the English phrase of the next line.
```suggestion
```